rusk/src/lib/services/prover.rs

```diff
@@ -80,7 +80,7 @@ impl RuskProver {
             })?;
 
         let mut pi: Vec<rusk_abi::PublicInput> =
-            Vec::with_capacity(5 + inputs.len() + 2 * outputs.len());
+            Vec::with_capacity(7 + inputs.len());
```

Comment:
If the output Len is now fixed to 2, shouldn't this be `5 + inputs.len() + 2 * 2` => `9 + inputs.len()`?

Reply:
https://github.com/dusk-network/rusk/pull/694/commits/4e39f0b16afc9243cceedf047ee71fafd98f3c4b